Sources/Fuzzilli/Lifting/JavaScriptLifter.swift

```diff
@@ -40,6 +40,10 @@ public class JavaScriptLifter: ComponentBase, Lifter {
     /// For node.js this is "global", otherwise probably just "this".
     let globalObjectIdentifier: String
 
+    /// Special properties which significantly change variable type
+    /// We should recollect type if we change them
+    let specialProperties = ["__proto__"]
```

Comment:
I'm not really happy that the Lifter is now responsible for deciding when to recollect type information. But we can worry about this later (and if other cases come up). In the simplest case, we can just create another Analyzer for that task :)